install/kubernetes/helm/istio/values.yaml

```diff
@@ -31,6 +31,25 @@ mixer:
     enabled: true
   telemetry:
     enabled: true
+  # mixer load shedding configuration.
+  # When mixer detects that it is overloaded, it starts rejecting grpc requests.
+  loadshedding:
+    # disabled, logonly or enforce
+    mode: enforce
```

Comment:
does enforce mean it will starts reject grpc requests when mixer is overladed?   And when this is happened, will the call from service A to B allowed?  (or not allowed because mixer is too busy)?

Sorry we could use a bit more explanation here.  Pls note all these comments are going to be in our istio.io doc to explain each of these params in values.yaml.